### PR TITLE
Corrected script for some item #11

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: Implementations + corrections of a bunch of items

### CODE REVIEW

**Comment by Akkarinage** [comment]
I've converted this Pull Request to a Draft until the "todo" comments you've introduced have been resolved.
Please also provide your source for this information.

Note to PR submitters: Huge PRs can cause more problems than they solve; listing what has been changed becomes difficult, risk of merge conflicts increases, risk of overlap from another PR increases, etc.

File: db/re/item_db_equip.yml [hunk 1]
```diff
@@ -162795,6 +163374,44 @@ Body:
...
+      Right_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 100
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      autobonus "{ bonus bUseSPrate,-100; bonus2 bSPRegenRate,100,1000; }",20,5000,BF_WEAPON;
+      autobonus "{ bonus bUseSPrate,-100; bonus2 bSPRegenRate,100,1000; }",10,5000,BF_MAGIC;
+      bonus bAtkRate,10*.@r;
+      bonus bMAtkRate,5*.@r;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:163405]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 2]
```diff
@@ -165057,6 +165674,81 @@ Body:
...
```
Comment (Atemo) [inline db/re/item_db_equip.yml:165708]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 3]
```diff
@@ -165057,6 +165674,81 @@ Body:
...
```
Comment (Atemo) [inline db/re/item_db_equip.yml:165742]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 4]
```diff
@@ -168103,6 +168869,42 @@ Body:
...
```
Comment (Atemo) [inline db/re/item_db_equip.yml:168899]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 5]
```diff
@@ -168925,6 +169727,42 @@ Body:
...
```
Comment (Atemo) [inline db/re/item_db_equip.yml:169757]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 6]
```diff
@@ -169998,6 +170836,47 @@ Body:
...
```
Comment (Atemo) [inline db/re/item_db_equip.yml:170871]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 7]
```diff
@@ -170848,6 +171727,42 @@ Body:
...
+      Both_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 100
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      autobonus "{ bonus bUseSPrate,-100; bonus2 bSPRegenRate,100,1000; }",50,5000,BF_WEAPON;
+      autobonus "{ bonus bUseSPrate,-100; bonus2 bSPRegenRate,100,1000; }",30,5000,BF_MAGIC;
+      bonus bAtkRate,10*.@r;
+      bonus bMAtkRate,5*.@r;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:171757]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 8]
```diff
@@ -171768,6 +172683,76 @@ Body:
...
+      Both_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 100
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      autobonus "{ bonus bUseSPrate,-100; bonus2 bSPRegenRate,100,1000; }",20,5000,BF_WEAPON;
+      autobonus "{ bonus bUseSPrate,-100; bonus2 bSPRegenRate,100,1000; }",10,5000,BF_MAGIC;
+      bonus bAtkRate,10*.@r;
+      bonus bMAtkRate,5*.@r;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:172747]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 9]
```diff
@@ -172401,6 +173386,44 @@ Body:
...
+      Both_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 100
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      autobonus "{ bonus bUseSPrate,-100; bonus2 bSPRegenRate,100,1000; }",50,5000,BF_WEAPON;
+      autobonus "{ bonus bUseSPrate,-100; bonus2 bSPRegenRate,100,1000; }",30,5000,BF_MAGIC;
+      bonus bAtkRate,10*.@r;
+      bonus bMAtkRate,5*.@r;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:173418]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 10]
```diff
@@ -172933,6 +173956,42 @@ Body:
...
```
Comment (Atemo) [inline db/re/item_db_equip.yml:173986]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 11]
```diff
@@ -173906,6 +174965,47 @@ Body:
...
```
Comment (Atemo) [inline db/re/item_db_equip.yml:174999]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 12]
```diff
@@ -174396,6 +175496,41 @@ Body:
...
+      Both_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 100
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      autobonus "{ bonus bUseSPrate,-100; bonus2 bSPRegenRate,100,1000; }",20,5000,BF_WEAPON;
+      autobonus "{ bonus bUseSPrate,-100; bonus2 bSPRegenRate,100,1000; }",10,5000,BF_MAGIC;
+      bonus bAtkRate,10*.@r;
+      bonus bMAtkRate,5*.@r;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:175525]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 13]
```diff
@@ -176088,6 +177223,43 @@ Body:
...
```
Comment (Atemo) [inline db/re/item_db_equip.yml:177254]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 14]
```diff
@@ -176552,6 +177724,37 @@ Body:
...
```
Comment (Atemo) [inline db/re/item_db_equip.yml:177749]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 15]
```diff
@@ -176966,6 +178169,40 @@ Body:
...
+      Both_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 100
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      autobonus "{ bonus bUseSPrate,-100; bonus2 bSPRegenRate,100,1000; }",30,5000,BF_WEAPON;
+      autobonus "{ bonus bUseSPrate,-100; bonus2 bSPRegenRate,100,1000; }",15,5000,BF_MAGIC;
+      bonus bAtkRate,10*.@r;
+      bonus bMAtkRate,5*.@r;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:178194]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 16]
```diff
@@ -177343,6 +178580,40 @@ Body:
...
+      Both_Hand: true
+    WeaponLevel: 4
+    EquipLevelMin: 100
+    Refineable: true
+    Script: |
+      .@r = getrefine();
+      autobonus "{ bonus bUseSPrate,-100; bonus2 bSPRegenRate,100,1000; }",20,5000,BF_WEAPON;
+      autobonus "{ bonus bUseSPrate,-100; bonus2 bSPRegenRate,100,1000; }",10,5000,BF_MAGIC;
+      bonus bAtkRate,10*.@r;
+      bonus bMAtkRate,5*.@r;
```
Comment (Atemo) [inline db/re/item_db_equip.yml:178605]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 17]
```diff
@@ -177702,6 +178973,40 @@ Body:
...
```
Comment (Atemo) [inline db/re/item_db_equip.yml:178998]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_equip.yml [hunk 18]
```diff
@@ -178085,6 +179390,40 @@ Body:
...
```
Comment (Atemo) [inline db/re/item_db_equip.yml:179415]:
```suggestion
      bonus bMatkRate,5*.@r;
```

File: db/re/item_db_usable.yml [hunk 19]
```diff
@@ -51183,6 +51183,16 @@ Body:
...
+  - Id: 100848
+    AegisName: Watermelon_Pudding_J
+    Name: Watermelon Pudding    # !todo check english name
+    Type: Healing
+    Weight: 30
+    Script: |
+      percentheal 3,3;
+      specialeffect2 999;
+      specialeffect2 572;
+      /* TODO check effect */
```
Comment (Atemo) [inline db/re/item_db_usable.yml:51195]: `NPC売却・倉庫への移動のみ可能です` I wonder if we should follow the description

**Review by Atemo — COMMENTED** [review]
Good work!

**Comment by Atemo** [comment]
The "todo" can (will?) be resolved later as it doesn"t change the effect of the item

File: db/re/item_db_equip.yml (hunk 1 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml:163405]: Thanks!

File: db/re/item_db_equip.yml (hunk 2 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 3 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 4 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 5 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 6 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 7 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 8 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 9 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 10 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 11 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 12 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 13 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 14 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 15 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 16 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 17 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_equip.yml (hunk 18 above)
Comment (idk-whoami) [reply db/re/item_db_equip.yml]: Thanks!

File: db/re/item_db_usable.yml (hunk 19 above)
Comment (idk-whoami) [reply db/re/item_db_usable.yml:51195]: follow the description of divine-pride and let's leave it like that for now

Thanks!

**Comment by Atemo** [comment]
Thanks!

**Pull request merged**